right/src/usb_report_updater.c

```diff
@@ -311,34 +311,40 @@ static void applyKeyAction(key_state_t *keyState, key_action_t *action, uint8_t
 
         switch (action->type) {
             case KeyActionType_Keystroke:
+            {
+                bool stickyModifiersUnChanged = true;
```

Comment:
It'd be easy to comprehend a non-negated variable like `stickyModifiersChanged`.

Reply:
Sure, however result is having to negate its every use further down the road so I am not sure if it is much of an improvement...